test/helpers.py

```diff
@@ -0,0 +1,19 @@
+######################################################################
+#
+# File: test/helpers.py
+#
+# Copyright 2023 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+import platform
+
+import pytest
+
+
+def skip_on_windows(*args, reason='Not supported on Windows', **kwargs):
```

Comment:
this thing is already in master

Reply:
yes, it was just moved around in this PR